types/manager-types.yaml

```diff
@@ -807,6 +819,22 @@ node_types:
   manager.nodes.ManagerIPSetter:
     derived_from: cloudify.nodes.ApplicationModule
     properties:
+      os_user:
+        description: IP setter's OS username. Will be created if not existing, and used to run the service.
+        type: string
+        default: cfyipsetter
+      os_group:
+        description: IP setter's OS group. Will be created if not existing, and used to run the service.
+        type: string
+        default: cfyipsetter
+      os_homedir:
+        description: IP setter's OS user's home directory. Will not be created if missing.
+        type: string
+        default: /etc/cloudify
+      sudoers_include_dir:
```

Comment:
Potentially, yes. I'm not sure how many we're going to need to add on this run though, so adding extra complexity and retrieving the sudoers include dir path from a relationship is probably overkill until we add another one?